modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/20_analyzers.yml

```diff
@@ -901,6 +901,16 @@
     - length: { tokens: 1 }
     - match:  { tokens.0.token: خورد }
 
+    - do:
```

Comment:
Is this to test stemming?

Reply:
Yes

Reply:
Then this test will need to be guarded with skip version or something. ES 9 will have mixed cluster tests with 8.last & 9.current and the 8.last won't have the stemming automatically correct?

Reply:
I would prefer a new test that is guarded, that way the original test isn't always skipped.

Reply:
All that would have made sense to me a few months ago. Now we live in a world where we merge Lucene 10 to "main" at some point which not necessarily is the point in which it becomes 9.0. So this is becoming a bit of a head-scratcher for me. I need to figure out if we can skip based on IndexVersion or not (since that is what we condition the behavioural change on), if we need some new sort of capability voodoo for that etc...

Reply:
fwiw I'm afraid I might have to introduce a "cluster_feature" for this. Maybe it makes sense to have one for "Cluster runs with Lucene 10".